app/assets/stylesheets/application.css

```diff
@@ -3,6 +3,8 @@
  * and any sub-directories. You're free to add application-wide styles to this file and they'll appear at
  * the top of the compiled file, but it's generally better to create a new file per style scope.
  *= require_self
- *= require main
- *= require_tree ./vendor
 */
+
+body {
+  background: pink;
```

Comment:
Because why not

Reply:
but for what? to test or sth?

Reply:
Just left there as a todo. We need to style all that stuff

Reply:
ok. we need to share some of the styles.